controllers/grid/queries/form/QueryForm.inc.php

```diff
@@ -77,8 +77,11 @@ function __construct($request, $assocType, $assocId, $stageId, $queryId = null)
 
 		$this->setQuery($query);
 
+		$user = $request->getUser();
 		// Validation checks for this form
-		$this->addCheck(new FormValidator($this, 'users', 'required', 'stageParticipants.notify.warning'));
+		$this->addCheck(new FormValidatorCustom($this, 'users', 'required', 'stageParticipants.notify.warning', function($users) use ($user) {
+			return in_array($user->getId(), $users) && count($users) > 1;
```

Comment:
I'd suggest removing the first part of the check -- it might be valid for e.g. an Editor to create a conversation with an Author and a Copyeditor, but not want to participate in it themselves.

Reply:
Thanks and done.